package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
this was probably an accident and should be deleted? I don't think we use node at all in this project